plugins/woocommerce/client/legacy/js/frontend/add-to-cart.js

```diff
@@ -207,6 +218,55 @@ jQuery( function( $ ) {
 		}
 	};
 
+	/**
+	 * Update cart live region message after add/remove cart events.
+	 */
+	AddToCartHandler.prototype.alertCartUpdated = function( e, fragments, cart_hash, $button ) {
+		var message = $button.data( 'success_message' );
```

Comment:
This part breaks the Add to cart functionality for extensions that are calling this function with the 4th parameter. We received reports of this after the release of WooCommerce 9.3.

Reply:
Hey @ralucaStan 

Would you mind sending me the link to the issue so I can better understand the regression?

Thanks!

Reply:
@Manussakis we have addressed it in a follow up PR: https://github.com/woocommerce/woocommerce/pull/51449

It turns out 3PD extensions will trigger the event manually sometimes and won't pass the button element in those cases. I put a fuller description on the PR.

Reply:
Thank you, @samueljseay !